ansible/roles/stack-sunbird/templates/sunbird_nodebb.env

```diff
@@ -1,10 +1,17 @@
 # Initialization variable for mongo
 # Ref: https://community.nodebb.org/topic/4325/set-up-nodebb-via-environment-variables
-database=mongo
+database=redis
+# redis database number
+redis_database={{nodebb_redis_database}}
+# declearing js heap memory size
+NODE_OPTIONS={{nodebb_heap_size|default('2048')}}
```

Comment:
Please reduce default heap size to 1024. Is there a problem with 1024?
All the pods in other env's have a maximum pod memory set to 1GB. So if we increase heap to more than 1GB, pod will crash if it tries to use more heap.

Reply:
in load test env we face this heap memory issue with 1 GB after running soak test for 4hr with 45L samples 
that's why we use 2GB there so I am using same here (2GB)

Reply:
replaced with 1GB